src/bindings/swig/python/tests/test_keyset.py

```diff
@@ -113,11 +120,18 @@ def test_python_copy(self):
 		ks = copy.copy(self.ks)
 		self.assertEqual(len(ks), len(self.ks))
 		self.assertEqual(ks[0].getKey(), self.ks[0].getKey())
+		self.assertTrue(ks == self.ks)
 
 		# deep copy
 		ks = copy.deepcopy(self.ks)
 		self.assertEqual(len(ks), len(self.ks))
 		self.assertNotEqual(ks[0].getKey(), self.ks[0].getKey())
+		self.assertTrue(ks == self.ks)
+
+	def test_helpers(self):
```

Comment:
Duplicate of test cases? (See below)

Reply:
What do you mean? The other file is python2

Reply:
Ahh ok, I did not expect that you will update python2 as we discussed in the other issue to remove it.